java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/EncryptingEntityTest.java

```diff
@@ -116,6 +120,89 @@ public void canSurviveNetworkFailuresInAesCbc() throws Exception {
         canBeWrittenIdempotently(AesCbcCipherDetails.INSTANCE_128_BIT);
     }
 
+    /* Cipher-generic tests */
+
+    private void canBeWrittenIdempotently(final SupportedCipherDetails cipherDetails) throws Exception {
+        final SecretKey secretKey = SecretKeyUtils.generate(cipherDetails);
+        final String content = RandomStringUtils.randomAlphanumeric(RandomUtils.nextInt(500, 1500));
+        final ExposedStringEntity contentEntity = new ExposedStringEntity(
+                content,
+                StandardCharsets.UTF_8);
+
+        final ByteArrayOutputStream referenceEncrypted = new ByteArrayOutputStream(content.length());
+        {
```

Comment:
I'm not following what this bracket signifies.

Reply:
This test was introduced with #296 and moved in this PR when I noticed it was below the test utility methods. The reason for the bracket was to isolate variable scopes, by the time I wrote this test I had been bitten by variable naming that made some tests seem to pass when they were supposed to fail and vice-versa. The braces made mistakenly-reused variables from the first round of encryption into compiler errors so I could be sure I wasn't mixing objects between encryption operations.